deploys/demo-test.syncfor.science/config/defaults/clients.json

```diff
@@ -1,50 +0,0 @@
-[{
```

Comment:
Can you clarify where this information lives now,instead of these configuration files?